xbmc/settings/dialogs/GUIDialogContentSettings.h

```diff
@@ -76,6 +74,10 @@ class CGUIDialogContentSettings : public CGUIDialogSettingsManualBase
   void InitializeSettings() override;
 
 private:
+  void setLabel2(const std::string &settingid, const std::string &label);
+  void toggleState(const std::string &settingid, bool enabled);
+  void setFocus(const std::string &settingid);
```

Comment:
As we do not say for public methods, I thought it's clear enough that we mean ALL. Feel free to adjust it in the wiki.